tango_simlib/tango_sim_generator.py

```diff
@@ -43,6 +44,8 @@
         'argoutDescription': 'doc_out',
         'argoutType': 'dtype_out'}
 
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
I see many recommendations not to do this (links below).  Is it really the only way?  Can we not do something at the point of failure, e.g. optionally switch to UTF-8, if a particular string causes a problem?  E.g. see a recent PR in KATCP:  https://github.com/ska-sa/katcp-python/pull/193

https://anonbadger.wordpress.com/2015/06/16/why-sys-setdefaultencoding-will-break-code/
https://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script